src/debuginfo.cpp

```diff
@@ -49,7 +49,7 @@ using namespace llvm;
 #include "julia_internal.h"
 #include "codegen_internal.h"
 #ifdef _OS_LINUX_
-#  define UNW_LOCAL_ONLY
+//#  define UNW_LOCAL_ONLY
```

Comment:
(note this is probably not needed here, I'm not sure anymore I'll have to check)